Gemfile.lock

```diff
@@ -53,6 +54,7 @@ GEM
     parallel (1.13.0)
     parallel_tests (2.9.0)
       parallel
+    public_suffix (4.0.7)
```

Comment:
Build is failing because:

```
public_suffix-4.0.7 requires ruby version >= 2.3, which is incompatible with the
current version, ruby 1.9.3p550
```